network/wsNetwork.go

```diff
@@ -113,6 +113,9 @@ var networkOutgoingConnections = metrics.MakeGauge(metrics.NetworkOutgoingConnec
 var networkIncomingBufferMicros = metrics.MakeCounter(metrics.MetricName{Name: "algod_network_rx_buffer_micros_total", Description: "microseconds spent by incoming messages on the receive buffer"})
 var networkHandleMicros = metrics.MakeCounter(metrics.MetricName{Name: "algod_network_rx_handle_micros_total", Description: "microseconds spent by protocol handlers in the receive thread"})
 
+var networkHandleMicrosByTag *metrics.TagCounter
```

Comment:
maybe to define it next to `networkMessageSentByTag` in wsPeer.go where they are all initialized?